src/Tasks/Microsoft.NET.Build.Tasks/ResolvePublishAssemblies.cs

```diff
@@ -32,7 +33,9 @@ public class ResolvePublishAssemblies : TaskBase
 
         public bool PreserveCacheLayout { get; set; }
 
-        public string FilterProjectAssetsFile { get; set; }
+        public bool DoNotTrackPackageAsResolved { get; set; }
+
+        public string FilterProjFiles { get; set; }
```

Comment:
These are specified via command line in msbuild like, hence the need to split by ";"
msbuild  /t:publish /p:FilterProjFile=profil1.xml%3bproflie.xml test.proj

If there is a good way to to specify an intemgroup [ i am assuming that is what you mean by non-scalar msbuild input, do let me know]